File: JavaScript/0500_keyboard_row/solution.js
function keyboard_row(words) {
    const row1 = "qwertyuiop";
    const row2 = "asdfghjkl";
    const row3 = "zxcvbnm";

    let result = [];
    for (let word of words) {
        let count = 0;
        if (row1.includes(word[0].toLowerCase())) {
            for (let index = 0; index < word.length; index ++) {
                if (row1.includes(word[index].toLowerCase())) {
                    count ++;
                }
            }
        } else if (row2.includes(word[0].toLowerCase())) {
            for (let index = 0; index < word.length; index ++) {
                if (row2.includes(word[index].toLowerCase())) {
                    count ++;
                }
            }
        } else {
            for (let index = 0; index < word.length; index ++) {
                if (row3.includes(word[0].toLowerCase())) {
                    count ++;
                }
            }
        }
        if (count == word.length) {
            result.push(word);
        }
    }
    return result;
}

const words = ["Hello","Alaska","Dad","Peace"];
console.log(keyboard_row(words));

File: Python/0500_keyboard_row/solution.py
def keyboard_row(words):
    row1 = "qwertyuiop"
    row2 = "asdfghjkl"
    row3 = "zxcvbnm"
    
    result = list()
    for word in words:
        count = 0
        if word[0].lower() in row1:
            for index in range(len(word)):
                if word[index].lower() in row1:
                    count += 1
        elif word[0].lower() in row2:
            for index in range(len(word)):
                if word[index].lower() in row2:
                    count += 1
        else:
            for index in range(len(word)):
                if word[index].lower() in row3:
                    count += 1
        if count == len(word):
            result.append(word)
            
    return result

words = ["tyrip","praveen","qwerty","alak"]
print(keyboard_row(words))